mod-data-import-converter-storage-server/src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -40,16 +40,16 @@ public void postTenant(TenantAttributes entity, Map<String, String> headers, Han
       if (ar.failed()) {
         handlers.handle(ar);
       } else {
+        Future<List<String>> sampleData = setupTestData(DEFAULT_JOB_PROFILE_SQL, headers, context);
         if (!isLoadSample(entity)) {
           LOGGER.debug("Test data will not be initialized.");
           handlers.handle(ar);
```

Comment:
```suggestion
         sampleData.setHandler(event -> handlers.handle(ar));
```

Reply:
@KaterynaSenchenko Thank you for the explanation! I will do.